### PR TITLE
[Sophia]2nd task sass

### PR DESCRIPTION
**Pull request by SofiMat**

### CODE REVIEW

**Comment by SofiMat** [comment]
https://sofimat.github.io/ui-internship-lwo2/template-sass/index.html

**Comment by prezlyataa** [comment]
Please resolve conflict

**Comment by SofiMat** [comment]
> Please resolve conflict

done

File: template-sass/script.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  function tabsChange(id) {
+    const tabs = document.getElementsByClassName('tabs-text');
+    const selectedTab = document.getElementById(`tabs-text-${id}`);
+    for (let i = 1; i <= tabs.length; i++) {
+      const tab = document.getElementById(`tabs-text-${i}`);
+      const tabButton = document.getElementById(i)
+      if (selectedTab === tab) {
+        tab.classList.add('show');
+        tab.classList.remove('hidden');
```
Comment (AlyonaPianykh) [inline template-sass/script.js]: you can chain this functions ```tab.classList.add('show').remove('hidden')```

File: template-sass/scss/media/_phone.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      .enterprise-item {
+        margin: 0;
+      }
+    }
+    
+    .contact {
+      padding: 74px 50px 84px 50px;
+
+      .contact-form {
+
```
Comment (AlyonaPianykh) [inline template-sass/scss/media/_phone.scss]: Please don't leave two or more empty lines. There should me maximum 1 empty dividing line

File: template-sass/script.js (hunk 1 above)
Comment (SofiMat) [reply template-sass/script.js]: I've tried, it doesn't work as expected.

**Pull request merged**